site/docs/tasks/using-opa.md

```diff
@@ -3,17 +3,39 @@
 
 When OPA is used for data governance, it is deployed as a stand-alone service.  Policies are defined in rego and uploaded to OPA.
 
+## Fybrik Default Policies
+
+Fybrik <strong><em>denys by default </em></strong>any request if no rule is triggered. This behavior can be changed to <strong><em>allow by default</em></strong> by creating the following rule and upload it to OPA in methods described in this page:
+
+```yaml
+package dataapi.authz
+
+rule [{}] { true }
+```
+
+You can also add conditions like
+
+```yaml
+ rule[{}] {
+  // conditions here
+ }
+```
+
+That would make it allow only if the conditions match and no other rule said otherwise (e.g., no other rule said that we should redact a column).
```

Comment:
The verdict `allow` will be reached only if the conditions hold, and no other rule has been triggered, e.g. a rule requiring column redaction.

Reply:
Fix. Thanks